swarm/pss/prox_test.go

```diff
@@ -235,16 +213,12 @@ func testProxNetwork(t *testing.T) {
 	services := newProxServices(tstdata, true, handlerContextFuncs, tstdata.kademlias)
 	tstdata.sim = simulation.New(services)
 	defer tstdata.sim.Close()
-	err := tstdata.sim.UploadSnapshot(fmt.Sprintf("testdata/snapshot_%d.json", nodeCount))
-	if err != nil {
-		t.Fatal(err)
-	}
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*120)
 	defer cancel()
-	snap := readSnapshot(t, nodeCount)
-	err = tstdata.sim.WaitTillSnapshotRecreated(ctx, snap)
+	filename := fmt.Sprintf("testdata/snapshot_%d.json", nodeCount)
```

Comment:
this can be inlined in the line below

Reply:
same as above. i suggest we discuss it with other team members. 
@frncmx : what's your opinion?

Reply:
I'm happy with the explanatory variable. As I mentioned I would maybe even make a little function instead of a variable, thus avoiding this filename generation pattern scattered and repeated everywhere.